flutter/analysis_options.yaml

```diff
@@ -13,8 +13,7 @@ analyzer:
     deprecated_member_use_from_same_package: warning
     # ignore sentry/path on pubspec as we change it on deployment
     invalid_dependency: ignore
-  exclude:
-    - example/**
```

Comment:
Why is that removed?

Reply:
This allows the example to be analysed and linted. That's also why there are so much changes in the example, because I've fixed the warnings for the example.